src/it/resources/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/InputWhitespaceAroundBasic.java

```diff
@@ -6,33 +6,34 @@
  **/
 class InputWhitespaceAroundBasic
 {
-    /** warn **/
-    private int mVar1= 1; // warn
-    /** warn **/
-    private int mVar2 =1; // warn
+    private int mVar1= 1; // violation ''=' is not preceded with whitespace.'
+    private int mVar2 =1; // violation ''=' is not followed by whitespace.'
     /** Should be ok **/
     private int mVar3 = 1;
+    int xyz;       // multiple space between content and double slash.
+    int abc; //       multiple space between double slash and comment's text.
+    int pqr;       //     testing both.
 
     /** method **/
     void method1()
     {
         final int a = 1;
-        int b= 1; // warn
-        b= 1; // warn
-        b +=1; // warn
-        b -=- 1 + (+ b); // warn
+        int b= 1; // violation ''=' is not preceded with whitespace.'
+        b= 1; // violation ''=' is not preceded with whitespace.'
+        b +=1; // violation ''.*' is not followed by whitespace.'
```

Comment:
Let's avoid .*

Reply:
the `+` was causing mismatching in violation message so I had to use `.*` to prevent that

Reply:
Hhhhmm, can put `\` before +

Reply:
done, escaped all the required symbols using this trick